src/Umbraco.Infrastructure/Examine/ContentValueSetBuilder.cs

```diff
@@ -53,6 +53,11 @@ public override IEnumerable<ValueSet> GetValueSets(params IContent[] content)
                 scope.Complete();
             }
 
+            return GetValueSetsEnumerable(content, creatorIds, writerIds);
```

Comment:
Yep but there's still an issue with some of the TODO notes that I mention in my other PR. So while this may resolve the scope issue being captured (Does it? I didn't get a chance to test this), it doesn't solve the other issues mentioned since we will still be double enumerating things due to the groupby which means still eagerly doing the lookups in the same request thread when ideally the value sets are entirely built on a background thread so we can get the response back to the user asap. Also still need to fix the triple lookup that occurs for the content index as per my other TODOs. These all don't need to be fixed 'now' but would like to get them all fixed soon-ish.